data_steward/cdr_cleaner/cleaning_rules/drop_extreme_measurements.py

```diff
@@ -0,0 +1,115 @@
+"""
+Background
+
+Background: For physical measurements collected by the Program, warnings or limits exist for height, weight, and BMI;
+however, there are still some outliers in the dataset from before the addition of these limits (or in cases where limits
+are more relaxed). These outliers need to be dropped to improve data quality.
+
+Scope: Create the following cleaning rule (all Weights are in KILOGRAMS all Heights are in CENTIMETERS): In the
+measurements table, extreme values of height, weight, and BMI should be dropped along with related values for the
+participant.
+
+This is expected to drop a very small number of rows (less than 300 total) based on values in the current CDR.
+
+"""
+
+# Project imports
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+
+DELETE_HEIGHT_ROWS_QUERY = """
+--drop all height recrods out of bounds
+DELETE
+FROM `{project_id}.{dataset_id}.measurement`
+WHERE measurement_source_concept_id = 903133
+AND value_as_number NOT BETWEEN 90 AND 228
+UNION ALL
```

Comment:
I like this because I would've preferred this way too. However, BigQuery refuses to allow this. I typically check if a query is valid by pasting it into the BQ query editor and populating it with test datasets without actually running it. If I need to see the output of a query, I typically replace `delete/update/insert` statements with `select` and only run it after making sure they are all replaced.
There are other ways to achieve this though. `measurement_id` is a good marker since it is unique to each row, and  there can be a subquery that picks and unions the bad measurement_ids